mmcls/models/classifiers/base.py

```diff
@@ -129,9 +129,9 @@ def train_step(self, data, optimizer):
 
         Args:
             data (dict): The output of dataloader.
-            optimizer (:obj:`torch.optim.Optimizer` | dict): The optimizer of
-                runner is passed to ``train_step()``. This argument is unused
-                and reserved.
+            optimizer (:obj:`torch.optim.Optimizer` | dict | optional): The
```

Comment:
Well, the right usage should be
```suggestion
            optimizer (:obj:`torch.optim.Optimizer` | dict, optional): The
```
Because `optional` is not a type, but a kind of annotation.